src/style.css

```diff
@@ -0,0 +1,132 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  box-sizing: border-box;
+  color: rgba(6, 11, 53, 1);
+}
+
+body {
+  margin: 0;
+}
+
+:root {
+  --color: rgba(97, 96, 112, 1);
+  --button-color: rgba(0, 172, 220, 1);
+  --hover-button-color: rgba(255, 255, 255, 1);
+}
+
+.product-card {
+  padding: 32px 0 16px;
+  width: 198px;
+  background: var(--hover-button-color);
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.product-card__img {
+  margin: 0 19px 40px;
+}
+
+.card-img {
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__name {
+  margin: 0 16px 4px;
+}
+
+.name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.product-card__code {
+  margin: 0 84px 16px 16px;
+}
+
+.code {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color);
+}
+
+.product-card__reviews {
+  position: absolute;
+  top: 268px;
+  left: 129px;
+}
+
+.reviews {
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.stars {
+  display: flex;
+  margin: 0 17px 24px 16px;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url("images\star.svg");
+  background-repeat: no-repeat;
+}
+
+.stars .stars_star:nth-child(-n+4) {
+  background-image: url("images\star-active.svg");
+}
+
+.price {
```

Comment:
please use the proper class name
![image](https://user-images.githubusercontent.com/22824431/230068133-abf2413a-a9c6-4ae3-a066-2980e0101784.png)

Reply:
isn't fixed. It should be like card__price or something